main/inc/lib/template.lib.php

```diff
@@ -479,33 +480,50 @@ public function set_header($status)
 
     /**
      * Returns the sub-folder and filename for the given tpl file.
-     * If template not found in overrides/ or custom template folder, the
-     * default template will be used.
+     *
+     * If template not found in overrides/ or custom template folder, the default template will be used.
      *
      * @param string $name
      *
      * @return string
      */
-    public function get_template($name)
+    public static function findTemplateFilePath($name)
     {
+        $sysCodePath = api_get_path(SYS_CODE_PATH);
```

Comment:
Please use SYS_TEMPLATE_PATH here (and modify the corresponding usage of $sysCodePath with $sysTemplatePath)

Reply:
This is because, in the future, the template path is likely to move *out* of main/ (and into app/ or somewhere around there), and then it will be less painful to make the change if we do this change here now.